drivers/flash/Kconfig.mimxrt

```diff
@@ -0,0 +1,44 @@
+config SOC_FLASH_IMXRT
+	bool "Flash Driver for NXP iMXRT familly"
+	depends on SOC_SERIES_IMX_RT
+	select FLASH_HAS_PAGE_LAYOUT
+	select FLASH_HAS_DRIVER_ENABLED
+	select CODE_DATA_RELOCATION
+	help
+	  Driver for external flash over FlexSPI interface. Work for Hyperflash memory.
```

Comment:
Consistent indenting required here (this is causing the doc build error):
```

	help
	  Driver for external flash over FlexSPI interface. Works for Hyperflash memory.
	  This will enable CODE_DATA_RELOCATION, as RAM run is required while writing the
	  external flash. QSPI will be implemented in a future release.
```